stubs/WebOb/webob/cookies.pyi

```diff
@@ -0,0 +1,129 @@
+
+class Morsel(dict[str, Incomplete]):
+    name: Incomplete
+    value: Incomplete
+    def __init__(self, name, value) -> None: ...
+    path: Incomplete
+    domain: Incomplete
+    comment: Incomplete
+    expires: Incomplete
+    max_age: Incomplete
+    httponly: Incomplete
+    secure: Incomplete
+    samesite: Incomplete
+    def __setitem__(self, k, v) -> None: ...
+    def serialize(self, full: bool = ...): ...
+    def __str__(self, full: bool = ...) -> str: ...
+
+def make_cookie(
+    name,
+    value,
+    max_age: Incomplete | None = ...,
+    path: str = ...,
+    domain: Incomplete | None = ...,
+    secure: bool = ...,
+    httponly: bool = ...,
+    comment: Incomplete | None = ...,
+    samesite: Incomplete | None = ...,
+): ...
+
+class JSONSerializer:
+    def dumps(self, appstruct): ...
+    def loads(self, bstruct): ...
+
+class Base64Serializer:
+    serializer: Incomplete
+    def __init__(self, serializer: Incomplete | None = ...) -> None: ...
+    def dumps(self, appstruct): ...
+    def loads(self, bstruct): ...
+
+class SignedSerializer:
+    salt: Incomplete
+    secret: Incomplete
+    hashalg: Incomplete
+    salted_secret: Incomplete
+    digestmod: Incomplete
+    digest_size: Incomplete
+    serializer: Incomplete
+    def __init__(self, secret, salt, hashalg: str = ..., serializer: Incomplete | None = ...) -> None: ...
+    def dumps(self, appstruct): ...
+    def loads(self, bstruct): ...
+
+class CookieProfile:
+    cookie_name: Incomplete
+    secure: Incomplete
+    max_age: Incomplete
+    httponly: Incomplete
+    samesite: Incomplete
+    path: Incomplete
+    domains: Incomplete
+    serializer: Incomplete
+    request: Incomplete
+    def __init__(
+        self,
+        cookie_name,
+        secure: bool = ...,
+        max_age: Incomplete | None = ...,
+        httponly: Incomplete | None = ...,
+        samesite: Incomplete | None = ...,
+        path: str = ...,
+        domains: Incomplete | None = ...,
+        serializer: Incomplete | None = ...,
+    ) -> None: ...
+    def __call__(self, request): ...
+    def bind(self, request): ...
+    def get_value(self): ...
+    def set_cookies(self, response, value, domains=..., max_age=..., path=..., secure=..., httponly=..., samesite=...): ...
+    def get_headers(self, value, domains=..., max_age=..., path=..., secure=..., httponly=..., samesite=...): ...
+
+class SignedCookieProfile(CookieProfile):
+    secret: Incomplete
+    salt: Incomplete
+    hashalg: Incomplete
+    original_serializer: Incomplete
+    def __init__(
+        self,
+        secret,
+        salt,
+        cookie_name,
+        secure: bool = ...,
+        max_age: Incomplete | None = ...,
+        httponly: bool = ...,
+        samesite: Incomplete | None = ...,
+        path: str = ...,
+        domains: Incomplete | None = ...,
+        hashalg: str = ...,
+        serializer: Incomplete | None = ...,
+    ) -> None: ...
+    def bind(self, request): ...
```

Comment:
I should add type annotations for the Cookie API. I personally have never used it, but it does appear to contain some useful things.

Reply:
I did add some of the cookie annotations. The `CookieProfile` part of the API can be added in a follow-up PR.